2_Background.md

```diff
@@ -0,0 +1,15 @@
+﻿2. Background
+
+2.1. Versións anteriores
+
+Estamos ante o deseño dun novo xogo, que non parte de versións anteriores; polo tanto, personaxes, escenarios e código serán deseñados partindo de cero; non se reutilizará código.
+
+2.2. Motores de xogo.
+
+Trátase dun proxecto en 3D. Empregaremos o motor de xogo "Unity", que se axusta perfectamente ás necesidades do proxecto e no que xa temos cumprida experiencia deseñando outros xogos. Pola mesma razón, para o deseño e animación das personaxes empregaremos Maya ou Blender.
+
+2.3. Referencias a outros xogos.
+
+O aspecto visual lembra a xogos de xadrez, xa que o/a xogador/a terá ao seu dispor pezas semellantes ás do xadrez, pero que poden mudar os seus movementos. Así mesmo, disporá dun taboleiro de xogo que lembra ao do xadrez, pero pode ter distintos tamaños e formas, incluso zonas internas sen escaques polas que non se pode pasar (semellante a certas zonas de pantallas do Candy Crash).
+
+Por outra banda, a medida que se avanza no xogo, iranse conseguindo recompensas dun xeito parecido ao "Legends of Runeterra".
```

Comment:
Personalmente cambiaría esta liña:
"Por outra banda, a medida que se avanza no xogo, iranse conseguindo recompensas dun xeito parecido ao "Legends of Runeterra"."

Por algo como:
Por outra banda, unha das principales mecánicas do xogo é o seu compoñente *rogue like* no cal recibiredes recompensas a medida que gañades duelos, dun xeito parecido ao que pasa en *Legends of Runeterra*

Reply:
Ok. Grazas.